src/edu/csus/ecs/pc2/ui/SelectJudgementPaneNew.java

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
```

Comment:
If the only change is copyright, why is this in the PR?

Reply:
It should've been changed in the PR for I209, but it was already changed so changed here.